features/facebookImport/src/components/activitiesMinistory/activitiesMinistory.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from "react";
+
+import i18n from "../../i18n";
+
+import BarChartHorizontal from "../dataViz/barChartHorizontal.jsx";
+
+import { fillArray } from "../dataViz/utils.jsx";
+
+import "./activitiesMinistory.css";
+import "./datePicker.css";
+import "../tabs/tabs.css";
+
+const monthsAbbreviation = i18n.t("common:months.abbreviation").split(" ");
+
+//TODO let this support actual dates not only years
+const Datepicker = ({ year, yearRange, onYearChange }) => {
```

Comment:
Naming nitpick here and below: CamelCase would probably say this should be DatePicker.